tensorflow/core/lib/wav/wav_io.cc

```diff
@@ -294,8 +294,8 @@ Status DecodeLin16WaveAsFloatVector(const std::string& wav_string,
         "Bad bytes per sample in WAV header: Expected ",
         expected_bytes_per_sample, " but got ", bytes_per_sample);
   }
-  const uint32 expected_bytes_per_second = bytes_per_sample * *sample_rate;
-  if (bytes_per_second != expected_bytes_per_second) {
+  const uint64 expected_bytes_per_second = (uint64)bytes_per_sample * *sample_rate;
```

Comment:
Please use ```static_cast<uint64>(bytes_per_sample)``` for casting.

Similarly in the line below.

Reply:
@akuegel 
Fixed